### PR TITLE
Fix py-ppft source extension

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-ppft/package.py [hunk 1]
```diff
@@ -21,3 +22,13 @@ class PyPpft(PythonPackage):
     depends_on('py-setuptools@0.6:', type='build')
     depends_on('py-six@1.7.3:', type=('build', 'run'))
     depends_on('py-dill@0.2.6:', type=('build', 'run'))
+
+    def url_for_version(self, version):
+        url = self.list_url
+        if version >= Version('1.6.4.9'):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-ppft/package.py]: This version is not in Spack

Comment (iarspider) [reply var/spack/repos/builtin/packages/py-ppft/package.py]: Added

File: var/spack/repos/builtin/packages/py-ppft/package.py [hunk 2]
```diff
@@ -21,3 +22,13 @@ class PyPpft(PythonPackage):
...
     depends_on('py-dill@0.2.6:', type=('build', 'run'))
+
+    def url_for_version(self, version):
+        url = "https://pypi.io/packages/source/p/ppft/"
+        if version >= Version('1.6.4.9'):
+            url += 'ppft-{0}.tar.gz'
+        else:
+            url += 'ppft-{0}.zip'
+
+        url = url.format(str(version))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-ppft/package.py]:
```suggestion
        url = url.format(version)
```

File: var/spack/repos/builtin/packages/py-ppft/package.py [hunk 3]
```diff
@@ -21,3 +22,13 @@ class PyPpft(PythonPackage):
     depends_on('py-setuptools@0.6:', type='build')
     depends_on('py-six@1.7.3:', type=('build', 'run'))
     depends_on('py-dill@0.2.6:', type=('build', 'run'))
+
+    def url_for_version(self, version):
+        url = "https://pypi.io/packages/source/p/ppft/"
+        if version >= Version('1.6.4.9'):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-ppft/package.py]:
```suggestion
        if version >= Version('1.6.4.8'):
```

**Pull request merged**